nautobot/ipam/views.py

```diff
@@ -193,118 +191,11 @@ class RIRBulkImportView(generic.BulkImportView):
 
 
 class RIRBulkDeleteView(generic.BulkDeleteView):
-    queryset = RIR.objects.annotate(aggregate_count=count_related(Aggregate, "rir"))
+    queryset = RIR.objects.annotate(prefix_count=count_related(Prefix, "rir"))
     filterset = filters.RIRFilterSet
     table = tables.RIRTable
 
 
-#
-# Aggregates
-#
-
-
-class AggregateListView(generic.ObjectListView):
-    queryset = Aggregate.objects.annotate(
-        child_count=RawSQL(
-            "SELECT COUNT(*) FROM ipam_prefix "
-            "WHERE ipam_prefix.prefix_length >= ipam_aggregate.prefix_length "
-            "AND ipam_prefix.network >= ipam_aggregate.network "
-            "AND ipam_prefix.broadcast <= ipam_aggregate.broadcast",
-            (),
-        )
-    ).select_related("rir")
-    filterset = filters.AggregateFilterSet
-    filterset_form = forms.AggregateFilterForm
-    table = tables.AggregateDetailTable
-    template_name = "ipam/aggregate_list.html"
-
-    def extra_context(self):
-        ipv4_total = 0
-        ipv6_total = 0
-
-        for aggregate in self.queryset:
-            if aggregate.prefix.version == 6:
-                # Report equivalent /64s for IPv6 to keep things sane
-                ipv6_total += int(aggregate.prefix.size / 2**64)
-            else:
-                ipv4_total += aggregate.prefix.size
-
-        return {
-            "ipv4_total": ipv4_total,
-            "ipv6_total": ipv6_total,
-        }
```

Comment:
Would this make sense to add similar to the Prefix list view?

Reply:
Maybe another issue to add this back in? The sidebar got ripped out in 1.5 but adding the sidebar back in is probably going to require some redesign since there's no search form anymore. Ex:

### v1.4
![image](https://user-images.githubusercontent.com/75227981/222284453-7a92198b-ad30-412c-85b9-90d42d0c4e73.png)

### v2.0
![image](https://user-images.githubusercontent.com/75227981/222283797-a612a36b-fa70-4781-bd1c-17969aa5c82a.png)

Reply:
Seeing as we're completely overhauling the UI, I don't think we should spend too much time making the existing v1 UI vibe in v2 alpha.